src/plugins/plugin-grpc.js

```diff
@@ -469,7 +469,7 @@ function unpatchServer(server) {
 
 // # Exports
 
-var SUPPORTED_VERSIONS = '0.13 - 1';
+var SUPPORTED_VERSIONS = '0.13 - 1.4.0';
```

Comment:
You're restricting the semver range here.. was that your intention? You expect that we won't be able ot support 1.4.1?